specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -5685,6 +5877,157 @@
         }
       },
       "description": "The list usages operation response."
+    },
+    "CanMigrateInput": {
+      "type": "object",
+      "description": "Input for canMigrate operation.",
+      "properties": {
```

Comment:
seems we missed "MigrationSku" property in canMigrateInput as MigrateInput has "MigrationSku".